Remotion/Web/ClientScript/Scripts/Utilities.ts

```diff
@@ -200,6 +200,52 @@ class ArgumentUtility
     ArgumentUtility.CheckNotNull(name, value);
     ArgumentUtility.CheckTypeIsJQueryObject(name, value);
   };
+
+  // Resolves any provided css selector, checks the element for null and returns it.
+  public static ResolveCssSelectorOrElement<TElement extends Element> (selectorOrElement: CssSelectorOrElement<TElement>, context?: ParentNode): TElement
+  {
+    if (TypeUtility.IsString(selectorOrElement))
+    {
+      const queryContext = context ? context : document;
```

Comment:
A stylistic quirk: `window.document`. I try to avoid using single-word members, so I use `window.document` instead of `document`. The dot in the reference is just something that helps me when reading code that refers to global state.